swagger-spec/schema_responses/definition.yaml

```diff
@@ -0,0 +1,119 @@
+type: object
+title: Schema Responses
+required:
+  - type
+  - attributes
+properties:
+  id:
+    type: string
+    readOnly: true
+    description: 'The unique identifier of the Schema Response.'
+  type:
+    type: string
+    readOnly: false
+    description: 'The type identifier of the Schema Response (`schema-response`).'
+  attributes:
+    type: object
+    title: Attributes
+    readOnly: false
+    description: 'The properties of the Schema Response.'
+    properties:
+      revision_justification:
+        type: string
+        readOnly: false
+        description: 'A string representing the reason the Schema Response was updated.'
```

Comment:
I feel as though this sentence makes it seem like the Schema Response models are the things being changed, as opposed to the responses for a registration.
"A string representing the reason a new Schema Response was needed" seems a bit less misleading to me.